packages/webapp/pages/account/profile.tsx

```diff
@@ -83,6 +88,165 @@ const AccountProfilePage = (): ReactElement => {
     },
   );
 
+  const RenderForm = () => (
+    <form ref={formRef} id="submit-profile">
+      <AccountContentSection
+        className={{ heading: 'mt-0' }}
+        title="Profile Picture"
+        description="Upload a picture to make your profile stand out and let people recognize
+        your comments and contributions easily!"
+      >
+        <div className="relative mt-6 flex">
+          <ImageInput
+            id={imageId}
+            className={{
+              img: 'object-cover',
+              container: 'border-4 !border-background-default',
+            }}
+            initialValue={user.image}
+            hoverIcon={<CameraIcon size={IconSize.Large} />}
+            onChange={(_, file) => {
+              if (!file) {
+                return;
+              }
+
+              updateUserProfile({
+                image: file,
+              });
+            }}
+          />
```

Comment:
Since this is quite a bit component, maybe let's move this to a func so it can be read easily?